scripts/standardising.py

```diff
@@ -26,54 +29,73 @@
 from scripts.tile.tile_index import Bounds, get_bounds_from_name
 
 
+@dataclass
+class StandardisingConfig:
+    """Standardising configuration.
+    gdal_preset: gdal preset to use. See `gdal.gdal_preset.py`
+    source_epsg: current EPSG code of the source file
+    target_epsg: desired EPSG code of the output file
+    gsd: expected Ground Sample Distance in meters
+    cutline: path to the cutline file. Must be `.fgb` or `.geojson`
+    scale_to_resolution: scale TIFFs to the specified x,y resolution. Defaults to None = no scaling.
+    """
+
+    gdal_preset: str
+    source_epsg: int
+    target_epsg: int
+    gsd: Decimal
+    create_footprints: bool
+    cutline: str | None
+    scale_to_resolution: list[Decimal] | None = None
+
+    def __post_init__(self) -> None:
+        if self.cutline is not None and not self.cutline.endswith((".fgb", ".geojson")):
+            raise ValueError(f"Only .fgb or .geojson cutlines are supported: {self.cutline}")
+        if self.scale_to_resolution is not None and len(self.scale_to_resolution) != 2:
+            raise ValueError(f"Scale_to_resolution must be exactly two items [xres, yres]: {self.scale_to_resolution}")
+
+
 def run_standardising(
-    todo: list[TileFiles],
-    preset: str,
-    cutline: str | None,
+    files_to_process: list[TileFiles],
```

Comment:
Tiles might be even more correct?

```suggestion
    tiles_to_process: list[TileFiles],
```